ash/src/extensions/khr/video_encode_queue.rs

```diff
@@ -0,0 +1,84 @@
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_video_encode_queue.html>
+
+use core::mem;
+use core::ptr;
+
```

Comment:
```suggestion
```

We've always let `rustfmt` sort these together. If we wish to diverge, we should do so across the codebase (at the very least your `video_queue.rs` :grimacing:).